src/aind_metadata_service/server.py

```diff
@@ -44,16 +54,39 @@
 # TODO: Handle favicon better?
 favicon_path = os.getenv("FAVICON_PATH")
 
+perfusions_smart_sheet_client = SmartSheetClient(
+    smartsheet_settings=perfusions_smartsheet_settings
+)
+
+funding_smart_sheet_client = SmartSheetClient(
+    smartsheet_settings=funding_smartsheet_settings
+)
+
+
+@app.get("/perfusions/{subject_id}")
+async def retrieve_perfusions(subject_id, pickle: bool = False):
+    """Retrieves perfusion information from smartsheet"""
+
+    # TODO: We can probably cache perfusions sheet
+    mapper = PerfusionsMapper(smart_sheet_client=perfusions_smart_sheet_client)
+    model_response = await run_in_threadpool(
+        mapper.get_model_response, input_id=subject_id
+    )
+    if pickle:
+        return model_response.map_to_pickled_response()
+    else:
+        return model_response.map_to_json_response()
+
 
 @app.get("/funding/{project_id}")
 async def retrieve_funding(project_id, pickle: bool = False):
     """Retrieves funding information from smartsheet"""
-    smart_sheet_client = SmartSheetClient(
-        smartsheet_settings=funding_smartsheet_settings
-    )
+
     # TODO: We can probably cache funding sheet
-    mapper = FundingMapper(smart_sheet_client=smart_sheet_client)
-    model_response = mapper.get_model_response(project_code=project_id)
+    mapper = FundingMapper(smart_sheet_client=funding_smart_sheet_client)
+    model_response = await run_in_threadpool(
```

Comment:
@dyf Does this seem reasonable? In my tests, there was some issue retrieving data from the funding sheet (it may have been moved), but waiting for a response was blocking calls elsewhere. This seems to work okay to alleviate the blocks.